src/components/OrderDetails/OrderDetails.tsx

```diff
@@ -18,17 +17,11 @@ export const OrderDetails = ({ order }: Props): ReactElement => {
         <span className="order_details__label">Status:</span>
         <span className="order_details__value">{order.status}</span>
       </div>
-      <div className="order_details__group">
-        <span className="order_details__label">Address:</span>
-        <div className="order_details__value">
-          <FormattedAddress address={order.address} />
-        </div>
-      </div>
       <div className="order_details__group order_details__group--items">
         <span className="order_details__label">Items:</span>
         <div id="order_details__items">
-          {order.items.length === 0 ? <p>There are no items in your order.</p> : null}
-          {order.items.map((item) => (
+          {!order.items?.length ? <p>There are no items in your order.</p> : null}
```

Comment:
if length is `-1` this will still be wrong since `!-1` is equal to falsy....
how about...
```tsx
{order.items?.length > 0 :  null  : <p>There are no items in your order.</p>}
```
since `null>0` and `0>0` and `-1>0` are all equal to `falsy`

Reply:
I didn't know a length could be -1? I'll change it

Reply:
Why would length be -1? @milkshakeuk

Reply:
I've pushed the changes @milkshakeuk